src/wranglerArtifactManager.test.ts

```diff
@@ -154,30 +157,50 @@ describe("wranglerArtifactsManager", () => {
 				expect(artifact).toEqual({
 					version: 1,
 					type: "version-upload",
+					version_id,
 					preview_url: "https://example.com",
 				});
-			}),
-				it("Skips artifact entries that are not parseable", async () => {
```

Comment:
This diff is slightly larger that necesary because these two were chained with `,` and I refactored them into two separate blocks with `;` and an empty line.